test/Nancy.Tests/Unit/Diagnostics/InteractiveDiagnosticsFixture.cs

```diff
@@ -3,6 +3,7 @@
     using Nancy.Diagnostics;
     using System.Collections.Generic;
     using System.Linq;
+    using System.Reflection;

```

Comment:
@thecodejunkie, you asked for me to point out any unresponded-to comments. I'm afraid you didn't see this one.